packages/ionic/src/controls/boolean/boolean-toggle-control.scss

```diff
@@ -0,0 +1,3 @@
+jsonforms-toggle-control {
```

Comment:
why do we need this?

Reply:
Currently not, I just added it for consistency reasons.